init.vim

```diff
@@ -20,6 +20,7 @@ scriptencoding utf-8
 let g:MAC = has('macunix')
 let g:LINUX = has('unix') && !has('macunix') && !has('win32unix')
 let g:WINDOWS = has('win32') || has('win64')
+let g:TMUX = !empty($TMUX)
```

Comment:
I think it would be better to put `let g:TMUX = !empty($TMUX)` in `core_config.vim`. Let's keep `init.vim` as simple as possible.